### ISSUE TITLE
MNIST 학습

### ISSUE THREAD

**Issue by jih3508** [Justin-A/DeepLearning101#2]
딥러닝대해서 공부하는 중인데
MNIST를 기존에 제공하는것 말고 제가 그린 이미지로 학습하고 싶은데 데이터를 불러와서 하는 방법을 알고 싶습니다. 아직 인터넷에  Tensorflow위주로 하는 것이 많아서 pytorch에 관한 정보가 잘 없는데......

**Comment by Justin-A** [comment]
안녕하세요, Justin입니다.
직접 그린 이미지를 이용하여 딥러닝 알고리즘을 학습하기 위해서는, pytorch custom dataset 키워드로 검색하셔서 자료를 확인하셔야 됩니다. 
이와 관련된 자료 중 쉽게 설명된 자료를 링크로 첨부해드립니다.
https://github.com/developer0hye/Custom-CNN-based-Image-Classification-in-PyTorch
감사합니다.

**Comment by jih3508** [comment]
cifa에 restnet152사용하는데 

![KakaoTalk_20201029_133813627](https://user-images.githubusercontent.com/48061617/97531024-a7098400-19f6-11eb-8caa-1d4eaa759f32.png)

test_loss값하고 test 정확도가 올라갔다 내려가는데 그럼 과적합이나 overffiting 같은 안 좋은 현상입니까?
그리고 loss값이 올라갔다가 내려가는데 그 이유도 제대로 알고 싶습니다.

**Comment by Justin-A** [comment]
안녕하세요, Justin입니다.
test_loss, test_accuracy 값들은 현재까지 학습이 진행된 모델이 실질적으로 분류를 잘 할 수 있는지 확인해볼 수 있는 지표입니다. train_loss 값이 줄어들지만 test_loss 값이 증가한 것은 현재 시점에서 과적합이라고 볼 수 있습니다. 

loss값이 올라갔다가 내려가는 이유는 해당 실험에서 설계한 학습 구조 내에서 Loss graph 값이 부분적으로 convex하기 때문에, gradient 값을 통해 weights값을 업데이트 할 때 학습률이 상대적으로 높아 loss값이 증가할 수 있습니다. 자세한 내용은 김성훈 교수님의 모두의 딥러닝 유튜브를 보시면 이해에 도움이 될 수 있습니다.
https://www.youtube.com/watch?v=TxIVr-nk1so&list=PLlMkM4tgfjnLSOjrEJN31gZATbcj_MpUm&index=6
감사합니다.
